avc/avcdecoderconfig_test.go

```diff
@@ -48,6 +48,80 @@ func TestAvcDecoderConfigRecord(t *testing.T) {
 	}
 }
 
+func TestAvcDecoderConfigRecordWithExtraBytes(t *testing.T) {
+	byteData, _ := hex.DecodeString("014d4029ffe10026674d40299e5281e022fde028404040500000030010000003032e04000ea6000057e43f13e0a001000468ef752000")
```

Comment:
This line is too long according to the linter, so I need to either break it up or add a //nolint:lll line before it.